superset/viz.py

```diff
@@ -935,7 +946,10 @@ def to_series(self, df, classed='', title_suffix=''):
             ys = series[name]
             if df[name].dtype.kind not in 'biufc':
                 continue
-            series_title = name
+            if isinstance(name, (list, tuple)):
```

Comment:
This hunk looks unrelated

Reply:
This is actually related. When you group by a numeric entity you can run into the same problem that the ID is to large to be displayed correctly in the legend or pop over. However we can just treat everything in the title a string since we can be sure its not used for anything mathematical.